plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/preferences/RunOptionsDefaultsComponent.java

```diff
@@ -141,6 +144,16 @@ public void onComplete(String contents) {
     messageTarget.setInfo("Set Pipeline Run Option Defaults");
   }
 
+  public RunOptionsDefaultsComponent(
+      Composite composite,
+      int numColumns,
+      DialogPageMessageTarget dialogPageMessageTarget,
+      DataflowPreferences prefs,
+      WizardPage page) {
+    this(composite, numColumns, dialogPageMessageTarget, prefs);
```

Comment:
If page is nullable (which it appears it is), I'd prefer to have all of the implementation in this constructor, and have the simpler constructor just pass in null for the `WizardPage` param.